lib/internal/cluster/master.js

```diff
@@ -85,7 +83,7 @@ cluster.setupMaster = function(options) {
     if (message.cmd !== 'NODE_DEBUG_ENABLED')
       return;
 
-    for (const worker of ObjectValues(cluster.workers)) {
+    for (const worker of cluster.workers.values()) {
```

Comment:
Can we keep using primordials, e.g.

```suggestion
    for (const worker of MapPrototypeValues(cluster.workers)) {
```

etc.?

Reply:
Is there any benefit (curious to know)?

Reply:
@yashLadha It makes this code work even if `Map.prototype` is modified, e.g. a user sets `Map.prototype.values` to a different function or deletes it.

Node.js core has moved towards using these cached primordials wherever necessary, for example we previously used `ObjectValues()` instead of `Object.values()` here.

Reply:
Nice, didn't know about the internals. Makes sense @addaleax . Changing this.

Reply:
Done @addaleax